.git-hooks/pre_commit/illegal_strings.rb

```diff
@@ -10,7 +10,7 @@ def run
         "\t" => "tab",
         "\r" => "carriage return",
         "\uFEFF" => "byte order marker (BOM)",
-        ("WCA " + "id") => "We prefer 'WCA ID', see https://github.com/thewca/worldcubeassociation.org/issues/268",
+        "WCA id" => "We prefer 'WCA ID', see https://github.com/thewca/worldcubeassociation.org/issues/268",
```

Comment:
I think this will fail because it lints itself

Reply:
yeah, I laughed out loud when I saw that